packs/feats/favor-of-heaven.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "ancestry",
         "description": {
-            "value": "<p>You're one of Hei Feng's chosen people and have the authority of the Celestial Court on your side. Once per day, you can cast a 6th-rank @UUID[Compendium.pf2e.spells-srd.Item.Spirit Blast] as a divine innate spell. In addition, you gain resistance 5 to both good and evil damage.</p>"
```

Comment:
Not sure that we need to specify that holy and unholy is sanctified?

Reply:
I added it just in case - I can remove those mentions.